http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala

```diff
@@ -14,15 +14,14 @@ import org.broadinstitute.dsde.workbench.leonardo.config.Config.{appMonitorConfi
 import org.broadinstitute.dsde.workbench.leonardo.config.SamConfig
 import org.broadinstitute.dsde.workbench.leonardo.dao.{
   CbasDAO,
-  CbasUiDAO,
   CromwellDAO,
   HailBatchDAO,
   SamDAO,
   WdsDAO,
   WsmApiClientProvider,
   WsmDao
 }
-import org.broadinstitute.dsde.workbench.leonardo.db.{DbReference, KubernetesServiceDbQueries, SaveKubernetesCluster, _}
+import org.broadinstitute.dsde.workbench.leonardo.db._
```

Comment:
This seems unrelated to removing CBAS UI?

Reply:
Yes this was updated when I had to run `scalafmt`

Reply:
Fwiw I removed this entire file in my other PR. It was useful for testing aks apps early on, but I don’t think it works anymore.